packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -82,46 +88,77 @@ function getListboxOptionsFromL1Network(network: Chain) {
 
 function NetworkContainer({
   network,
+  balanceFor,
   children
 }: {
   network: Chain
+  balanceFor?: string
   children: React.ReactNode
 }) {
-  const { backgroundImage, backgroundClassName } = useMemo(() => {
-    const { isArbitrum, isArbitrumNova } = isNetwork(network.id)
+  const { backgroundImage, backgroundClassName, customAddressBannerClassName } =
+    useMemo(() => {
+      const { isArbitrum, isArbitrumNova } = isNetwork(network.id)
+
+      if (!isArbitrum) {
+        return {
+          backgroundImage: `url('/images/TransparentEthereumLogo.webp')`,
+          backgroundClassName: 'bg-eth-dark',
+          customAddressBannerClassName: 'bg-cyan border-eth-dark'
+        }
+      }
 
-    if (!isArbitrum) {
-      return {
-        backgroundImage: `url('/images/TransparentEthereumLogo.webp')`,
-        backgroundClassName: 'bg-eth-dark'
+      if (isArbitrumNova) {
+        return {
+          backgroundImage: `url('/images/ArbitrumNovaLogo.svg')`,
+          backgroundClassName: 'bg-arb-nova-dark',
+          customAddressBannerClassName: 'bg-orange border-arb-nova-dark'
+        }
       }
-    }
 
-    if (isArbitrumNova) {
       return {
-        backgroundImage: `url('/images/ArbitrumNovaLogo.svg')`,
-        backgroundClassName: 'bg-arb-nova-dark'
+        backgroundImage: `url('/images/ArbitrumOneLogo.svg')`,
+        backgroundClassName: 'bg-arb-one-dark',
+        customAddressBannerClassName: 'bg-cyan border-arb-one-dark'
       }
-    }
+    }, [network])
 
-    return {
-      backgroundImage: `url('/images/ArbitrumOneLogo.svg')`,
-      backgroundClassName: 'bg-arb-one-dark'
-    }
-  }, [network])
+  const showCustomAddressBanner = balanceFor && utils.isAddress(balanceFor)
 
   return (
-    <div
-      className={`relative rounded-xl p-1 transition-colors ${backgroundClassName}`}
-    >
+    <>
```

Comment:
This is the banner. Colors change base on network.

<img width="518" alt="Screenshot 2023-06-20 at 17 41 16" src="https://github.com/OffchainLabs/arbitrum-token-bridge/assets/47932951/1aaf535b-d902-40c2-9c77-439e714a1060">